.prettierignore

```diff
@@ -0,0 +1 @@
+./webapp/generated
```

Comment:
shouldn't this be `/webapp/src/shared/generated`?
I don't think it will find any folder name generated inside webapp folder, which is the intent, like this.